example/src/sdk/workflows/assets/init.ts

```diff
@@ -26,4 +27,12 @@ console.log(`${pretag} done`)
 console.log(`${pretag} link intermediate wallet with root wallet`)
 await register()
 console.log(`${pretag} done`)
-await createOrgNode(orgName)
```

Comment:
@anky2k have to fix this example code here because passing the org name to createOrgNode sdk function will not work

Reply:
hmm yea this needs to be a wallet address // actually function expects a string - infact the logic will not break within the function with jst call - it will break with successive calls - good catch - thanks